### DISCUSSION TITLE
avoid "nvm for windows" phrasing

### DISCUSSION THREAD

**Discussion by ljharb** [coreybutler/nvm-windows#762] [General]
The readme says "NVM for Windows", but nvm.sh does work on windows - in WSL2, cygwin, and git bash - so perhaps this project could describe itself as `nvm-windows`, to avoid confusion?

**Comment by leafac** [comment]
See https://github.com/coreybutler/nvm-windows/issues/757#issuecomment-1048932480

**Comment by coreybutler** [comment]
I know this has been a pain point - I feel it too... constantly. Adding another moniker will only add to the confusion though. 

Ultimately, there needs to be a distinctly different name. Renaming this project would cause even more confusion though. When this project was created, Windows didn't have WSL/WSL2... so within the Windows community, this tool has been around considerably longer than nvm. 8+ years of blogs, guides, articles, videos, etc. all refer to this as NVM4W or "NVM for Windows".

I am working on a new effort called rt (short for runtime), which will serve as a successor for this project (among other things). The original concept was announced in [the discussions](https://github.com/coreybutler/nvm-windows/discussions/565#discussioncomment-58112) in 2020 and has been referenced throughout the issues/wiki in this repository. A public announcement was made in [Node Weekly](https://nodeweekly.com/issues/407) in 2021. Point: I've been seeding the roots of transition for awhile.

**tldr;** The confusion will go away when a new project succeeds this one. My time is better spent focusing on that.

I'm currently aiming for a summer release of rt.

**Reply by ljharb** [reply]
That's fair, but `nvm-windows` is the repo name, so i'm not sure it'd really be a new term.
